### ISSUE TITLE
Release v1.30.16

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#1077]
:robot: A dedicated issue to run tests and deploy release [v1.30.16](https://github.com/deckhouse/deckhouse/milestone/24).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | gcp | azure | openstack | yandex-cloud | vsphere | static`
  - `git_ref` is a tag or branch: `vX.Y.Z | test-X.Y.Z* | main | release-X.Y`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `containerd | docker`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag or branch: `vX.Y.Z | test-vX.Y.Z* | main | release-X.Y`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build modules FE' and 'build FE' jobs are finished.

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [f68479](https://github.com/deckhouse/deckhouse/commit/f68479207c6ac853f9757143946673b71c49b472) in branch **main**:
```
Re-generate changelog (#1078)

Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag 'v1.30.16' is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [1119d6](https://github.com/deckhouse/deckhouse/commit/1119d63edac69bd9e0c64166755716247b5deb93) in branch **main**:
```
Re-generate changelog (#1075)

Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a5b037](https://github.com/deckhouse/deckhouse/commit/a5b0371acf01d4f2a94df07c61bca85f31b0b944) in branch **main**:
```
[docs] Fix page headers (#1063)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c5f3ac](https://github.com/deckhouse/deckhouse/commit/c5f3ac02c6099a867132e5917c7d7fa5df67ec5d) in branch **main**:
```
[flant-integration] validate kubernetes version migration is one-time migration (#1080)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [842882](https://github.com/deckhouse/deckhouse/commit/84288298a668340c5d206f08924d4f61dee87097) in branch **main**:
```
[ci] fix: changelog fail on new issue (#1087)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d8da8c](https://github.com/deckhouse/deckhouse/commit/d8da8c5b8a03425da99a95010cb766d9fe53c16f) in branch **main**:
```
[node-manager] Fix incorrect prometheus rules fo k8s 1.19. (#1081)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [564ddf](https://github.com/deckhouse/deckhouse/commit/564ddfa2497c65c3176e718e7c28beed463e345f) in branch **main**:
```
[node-manager] fixup configurable kubelet log rotation (#1079)

Co-authored-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```
